addons/godot_rl_agents/sensors/sensors_3d/GridSensor3D.gd

```diff
@@ -38,30 +38,47 @@ class_name GridSensor3D
 		cell_height = value
 		_update()	
 
-@export_range(1, 21, 2, "or_greater") var grid_size := 3:
-	get: return grid_size
+@export_range(1, 21, 2, "or_greater") var grid_size_x := 3:
```

Comment:
Allows different sizes for x and z axes. 
Not sure if necessary, I have only tested with equal size so far so this code change is untested.